ts/components/LabelledItem.tsx

```diff
@@ -35,6 +35,9 @@ const styles = StyleSheet.create({
   },
   bottomLine: {
     borderBottomWidth: 1
+  },
+  textInputMask: {
+    fontFamily: "Titillium Web"
```

Comment:
To let this style to be compliant with both supported OS, I would suggest to use the `makeFontFamilyName` function:

Please check it [here](https://github.com/pagopa/io-app/blob/master/ts/components/core/fonts.ts#L72)